src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -15,25 +15,29 @@ AbstractTableGenerator::AbstractTableGenerator(const std::shared_ptr<BenchmarkCo
 void AbstractTableGenerator::generate_and_store() {
   Timer timer;
 
-  std::cout << "- Loading/Generating tables " << std::flush;
+  std::cout << "- Loading/Generating tables " << std::endl;
   auto table_info_by_name = generate();
-  std::cout << "(" << format_duration(std::chrono::duration_cast<std::chrono::nanoseconds>(timer.lap())) << ")"
```

Comment:
?

Reply:
Load times printed for each table now. Want an aggregation, too?

Reply:
Yes. Remember that not every benchmark uses the FileBasedTableGenerator. Also, it helps to see how long I will have to wait the next time, and I don't want to sum up the times for that.

Reply:
ack